src/Screens/DiveLogsForms/index.tsx

```diff
@@ -259,112 +260,117 @@ const SimpleDiveLogsForms: FunctionComponent<
       render={({ values, form }) => {
         formRef.current = form;
         return (
-          <SafeAreaView style={styles.container}>
-            <ExitModal
-              subtext={t('diveLogForm.EXIT_MODAL_SUBTEXT_FOR_SIMPLE_FORM')}
-              isVisible={modalIsOpen}
-              modalAction={modalAction}
-              modalCancelAction={modalCancelAction}
-              actionText={t('EXIT')}
-              cancelActionText={t('CANCEL')}
-            />
-            <ScrollView
-              nestedScrollEnabled
-              keyboardShouldPersistTaps="handled"
-              style={[
-                styles.scrollContainer,
-                page !== stages.length && {
-                  marginBottom: isBelowHeightThreshold ? 100 : 80,
-                },
-              ]}>
-              <View style={styles.headerContainer}>
-                {page > 0 && page !== stages.length ? (
-                  <TouchableWithoutFeedback onPress={previous}>
+          <KeyboardAvoidingView
```

Comment:
most of the changes in this file are just whitespace changes. you can view the diff while ignoring whitespace changes like this: https://github.com/Zentacle/snorkel-app/pull/48/files?diff=split&w=1